libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2267,11 +2267,9 @@ void TypeChecker::endVisit(ElementaryTypeNameExpression const& _expr)
 
 void TypeChecker::endVisit(Literal const& _literal)
 {
-	bool const v050 = m_scope->sourceUnit().annotation().experimentalFeatures.count(ExperimentalFeature::V050);
-
 	if (_literal.looksLikeAddress())
 	{
-		// Assign type here if it even looks like an address. This prevents double error in 050 mode for invalid address
+		// Assign type here if it even looks like an address. This prevents double error for invalid addresses
```

Comment:
Well - if one wants to be *really* pedantic, then one should probably change to ``This prevents double errors for invalid addresses.`` - but I'm fine with it as it is, it's just a code comment after all...

Reply:
Fixed it and squashed.